### PR TITLE
two minor spelling fixes

### PR DESCRIPTION
**Pull request by paralax**

### CODE REVIEW

**Comment by leecalcote** [comment]
LGTM

File: whitepaper/README.md [hunk 1]
```diff
@@ -863,7 +863,7 @@ Serverless architectures provide an exciting new deployment option for cloud nat
...
+The CNCF Serverless Working Group, in partnership with [Redpoint Ventures](http://www.redpoint.com/), recently published a a [Serverless Landscape](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0). It illustrates some of the major serverless projects, tooling and services that are available in the ecosystem. It is not intended to represent a comprehensive, fully inclusive serverless ecosystem, nor is it an endorsement, rather just an overview of the landscape.. It is expected that owners of each will provide updates in an attempt to keep it up to date.
```
Comment (duglin) [inline whitepaper/README.md]: this is why I really prefer to wrap stuff at 80 columns - github doesn't show which word was changed.

**Comment by paralax** [comment]
Serverless was misspelled. 

J

**Comment by duglin** [comment]
LOL now that's funny

File: whitepaper/README.md (hunk 1 above)
Comment (leecalcote) [reply whitepaper/README.md]: Bug!!

Comment (duglin) [reply whitepaper/README.md]: true - but that's also reality :-)

**Pull request merged**